examples/crossmodel_relation.py

```diff
@@ -55,7 +55,7 @@ async def main():
 
         print('Deploying wordpress')
         application_2 = await consuming_model.deploy(
-            'cs:trusty/wordpress-5',
+            'ch:trusty/wordpress',
```

Comment:
interesting how we take this from trusty but use xenial series

Reply:
Yeah this example is not working in 3.0 (because of the wordpress charm), needs to be revisited. I added a TODO there to come back to it :+1: